dev/storage.h

```diff
@@ -947,28 +947,28 @@ namespace sqlite_orm {
             }
 
             template<class It, class Projection = polyfill::identity>
-            void insert_range(It from, It to, Projection project = {}) {
+            int64 insert_range(It from, It to, Projection project = {}) {
```

Comment:
It makes no sense why inserting a range has to return inserted id. I mean who needs to insert let's say 10 objects and then expect to get latest object's id. `RETURNING` feature is more appropriate here and once it is added your diff in this function will just removed. So let's just skip changing a lot of public APIs cause one person wants it. It is related to both `insert_range` overloads